### ISSUE TITLE
regression with --help-header and 2.4.2 release

### ISSUE THREAD

**Issue by vapier** [megastep/makeself#247]
the 2.4.0 and older releases work, but starting with 2.4.2, breaks our build

i bisected it to commit 58ff507611de976714278c123cf32a689ce4dc05.

**Comment by vapier** [comment]
hit enter a little too soon there.  here's the reproduction.

@pdietl should just revert this broken commit.

old version works fine.
```sh
$ echo "msg with 'quotes and spaces'" > msg

$ git checkout 58ff507611de976714278c123cf32a689ce4dc05^
Previous HEAD position was 58ff507611de Remove unnecessary escaped single quote, which throws off certain text editors with highlighting, like vim (#184)
HEAD is now at 6ca4d7af2f44 fix #180: edit .travis.yml to allow_failures on windows (#181)

$ ./makeself.sh -q --help-header ./msg test test.run test /bin/true
$ ./test.run
Verifying archive integrity...  100%   MD5 checksums are OK. All good.
Uncompressing test  100%
```

new version is broken.  notice the "command not found" errors.
```sh
$ git checkout 58ff507611de976714278c123cf32a689ce4dc05
Previous HEAD position was 6ca4d7af2f44 fix #180: edit .travis.yml to allow_failures on windows (#181)
HEAD is now at 58ff507611de Remove unnecessary escaped single quote, which throws off certain text editors with highlighting, like vim (#184)

$ ./makeself.sh -q --help-header ./msg test test.run test /bin/true
$ ./test.run 
./test.run: line 20: and: command not found
Verifying archive integrity...  100%   MD5 checksums are OK. All good.
Uncompressing test  100%
```

**Comment by pdietl** [comment]
@vapier my b.
